src/components/categoryFilter.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
```

Comment:
Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.